dev/tests/api-functional/testsuite/Magento/GraphQl/Ups/SetUpsShippingMethodsOnCartTest.php

```diff
@@ -62,13 +62,17 @@ protected function setUp()
     }
 
     /**
-     * @magentoApiDataFixture Magento/Checkout/_files/quote_with_address_saved.php
+     * @magentoApiDataFixture Magento/Customer/_files/customer.php
+     * @magentoApiDataFixture Magento/Catalog/_files/product_simple.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/customer/create_empty_cart.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/add_simple_product.php
+     * @magentoApiDataFixture Magento/GraphQl/Quote/_files/set_new_shipping_address.php
      * @magentoApiDataFixture Magento/Ups/_files/enable_ups_shipping_method.php
      */
     public function testSetUpsShippingMethod()
     {
         $quote = $this->quoteFactory->create();
-        $this->quoteResource->load($quote, 'test_order_1', 'reserved_order_id');
+        $this->quoteResource->load($quote, 'test_quote', 'reserved_order_id');
```

Comment:
Please, replace `quoteResource` on `\Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface`

Reply:
> Please, replace quoteResource on \Magento\Quote\Model\QuoteIdToMaskedQuoteIdInterface

**done**

Reply:
@naydav , all Ups methods have been added into test. Could you please review?

Thank you!